setup.py

```diff
@@ -64,26 +58,27 @@
         'Topic :: Scientific/Engineering',
         'Topic :: Software Development',
         'Topic :: Utilities'
-      ],
-      keywords='planet api sdk client',
-      author='Jennifer Reiber Kyle',
-      author_email='jennifer.kyle@planet.com',
-      url='https://github.com/planetlabs/planet-client-python',
-      license='Apache 2.0',
-      packages=find_packages(exclude=['examples', 'tests']),
-      data_files=[('', ['LICENSE'])],
-      include_package_data=True,
-      zip_safe=False,
-      python_requires='>=3.7',
-      install_requires=install_requires,
-      extras_require={
-          'test': test_requires,
-          'lint': lint_requires,
-          'docs': doc_requires,
-          'dev': test_requires + lint_requires + doc_requires},
-      entry_points={
-          'console_scripts': [
-              'planet=planet.cli.cli:main',
-          ],
-        },
-      )
+    ],
```

Comment:
This is the section that prompted the use of the modifier. Without it, these classifiers are clumped together one or two to a line. It's just difficult to understand in that format. [This](https://github.com/planetlabs/planet-client-python/pull/406/files#r819820037) is what happens without the modifier.